driver/config/config_test.go

```diff
@@ -553,7 +553,7 @@ func TestViperProvider_Defaults(t *testing.T) {
 			assert.False(t, p.SelfServiceFlowVerificationEnabled(ctx))
 			assert.True(t, p.SelfServiceStrategy(ctx, "password").Enabled)
 			assert.True(t, p.SelfServiceStrategy(ctx, "profile").Enabled)
-			assert.True(t, p.SelfServiceStrategy(ctx, "link").Enabled)
+			assert.True(t, p.SelfServiceStrategy(ctx, "code").Enabled)
```

Comment:
When changing tests, it's ok to add assertions, but very rarely ok to remove or modify existing assertions. Please add the link assertion back in here!

Reply:
You're right, I gotta internalize that sentiment. 

In this case, we are expecting a change in behavior, so changing the assertion to expect `False` should be correct, right?

Reply:
Yup!

Reply:
Updated the assertion [here](https://github.com/ory/kratos/pull/2645/files#diff-deb13d25cad45e9e2975fd0ad5710541fd5b7ae76411f9ce50c4af6a053f7593R558)